packages/types/src/product/common.ts

```diff
@@ -1101,9 +1100,9 @@ export interface CreateProductVariantDTO {
    */
   width?: number
   /**
-   * The product variant options to create and associate with the product variant.
+   * The product variant options to associate with the product variant.
    */
-  options?: CreateProductVariantOptionDTO[]
+  options?: Record<string, string>
```

Comment:
options can have multiple values if i remember correctly.
```suggestion
  options?: Record<string, string[]>
```

Reply:
@riqwan you can only have one option of a type, for example `color: green`. You can't have it both green and blue

Reply:
Ah sorry, I thought this was on the Product.